openmetadata-docs/content/deployment/upgrade/docker.md

```diff
@@ -22,39 +22,104 @@ It is adviced to go through [openmetadata release notes](/deployment/upgrade#bre
 
 </Warning>
 
-### 1. Download docker-compose.yaml file
-
-Go to [github.com/open-metadata/OpenMetadata/releases](https://github.com/open-metadata/OpenMetadata/releases). The latest release will be at the top of this page.
-
-Download the new `docker-compose.yml` file. You can run the below command to download the file (replacing `{version}` with the correct version) or simply visit the page and download the file from your browser.
-```
-wget https://github.com/open-metadata/OpenMetadata/releases/download/{version}-release/docker-compose.yml
-```
-or if you wish to use postgres as the database
-```
-wget https://github.com/open-metadata/OpenMetadata/releases/download/{version}-release/docker-compose-postgres.yml
-```
-
-### 2. Backup your Data [IMPORTANT]
-
-Make a backup of your database. You can find the steps to follow [here](/deployment/upgrade//deployment/backup-restore-metadata#backup-metadata). Please note this is an important step as it would allow you to revert to a stable state if any issues were to happen during your upgrade.
-
-### 3. Add Volumes and Publish Ports
-
-Update the docker compose file you just downloaded to add persistent volume for MySQL. You can follow the steps [here](/deployment/docker/volumes#docker-volumes). If you had previously configured volumes for MySQL and the ingestion container make sure the 0.13.0 compose file is referencing the correct volume so that 0.12.x data will be available when upgrading to 0.13.0.
-
-**Note:** in 0.13.0 we are not publishing MySQL ports to the host container. If you wish to do so you will need to update the MySQL service in the compose file to have the following configuration
-```yaml
-services:
-  mysql:
-    ...
+## Upgrade from 0.12.3 to 0.13.1
+
+Your production deployment should go from stable version to stable version. This translated to moving from 0.12.3 to 0.13.1 to
+get the latest stable OpenMetadata release.
+
+Database volumes were just introduced in the 0.13.0 feature release. This means that when moving from 0.12.3 to 0.13.1, we'll
+need to do some extra steps to ensure that all your metadata is safe and sound. In this section, we'll guide you step by step on
+what you'll need to do to:
+
+1. Backup your data,
+2. Add volumes and port mapping to the databases and restore your data,
+3. Start OpenMetadata with the 0.13.1 release.
+
+These steps are based on MySQL, but the steps are analogous for Postgres.
```

Comment:
I think we should specify that this step is "if you have not added volumes yourself". User may have decided to add volumes themselves by modifying the docker image.